transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/insight/AddTransferMethodInsightTest.java

```diff
@@ -98,5 +113,102 @@ public void testAddTransferMethod_verifyInsightEventCreatedOnAddTransferMethodLo
 
     }
 
+    @Test
+    public void testAddTransferMethod_verifyInsightEventCreatedOnClickCreateTransferMethod() {
+        mActivityTestRule.launchActivity(null);
+
+        onView(withId(R.id.add_transfer_method_button)).perform(nestedScrollTo(), click());
+
+        verify(mHyperwalletInsightMockRule.getInsight(),
+                timeout(5000).times(1)).trackClick(any(Context.class),
+                eq(AddTransferMethodActivity.TAG),
+                eq(PageGroups.TRANSFER_METHOD),
+                eq(HyperwalletInsight.LINK_SELECT_TRANSFER_METHOD_CREATE),
+                mParamsCaptor.capture());
+
+        assertThat(mParamsCaptor.getValue().get("hyperwallet_ea_country"), is("US"));
+        assertThat(mParamsCaptor.getValue().get("hyperwallet_ea_currency"), is("USD"));
+        assertThat(mParamsCaptor.getValue().get("hyperwallet_ea_type"), is("BANK_ACCOUNT"));
+        assertThat(mParamsCaptor.getValue().get("hyperwallet_ea_profile_type"), is("INDIVIDUAL"));
+    }
+
+    @Test
+    public void testAddTransferMethod_verifyInsightEventCreatedOnClickCreateTransferMethodWithBusinessError() {
```

Comment:
Rename to testAddTransferMethod_verifyInsightEventCreatedOnClickCreateTransferMethodWithApiError please